components/CategoryImage/CategoryImage.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
```

Comment:
This component doesn't seem to need `use client`? I've removed it and it still works.

Reply:
b7322b5f